src/main/resources/workflow/workflow.xml

```diff
@@ -23,9 +23,8 @@
               <arg name="class.name">com.atlassian.jira.workflow.function.issue.IssueReindexFunction</arg>
             </function>
             <function type="class">
-              <arg name="eventTypeId">1</arg>
-              <arg name="class.name">com.atlassian.jira.workflow.function.event.FireIssueEventFunction
-                            </arg>
```

Comment:
wouldn't the problem be the newline?

Reply:
I wasn't sure. I guessed that the `eventTypeId` should be the same as for the other transitions in the file?

Reply:
`1` is `Issue Created` and `13` is `Generic Event`

Reply:
Yeah, sorry, just looked properly at that Slack message...!